Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM centos:7
```

Comment:
Someone suggested to use fedora-minimal to have smaller images.

Reply:
ok, I take note of it and test it when I will be doing the openshift readme part